### ISSUE TITLE
App crashes when downloading a video 

### ISSUE THREAD

**Issue by Apevio** [christianselig/apollo-bugs#1375]
**Describe the bug**
When I try to download videos (not GIFS) the app will crash but I’ll still have the downloaded content. 

1. Find any video 
2. Press and hold on video 
3. Press “Download Video”
4. App goes through its process, then crash

**iOS version**: iOS 15 public beta 
**Apollo version**: 1.10.11
**iOS device**: IPhone XR
**Jailbroken**: No
**Reproducible:** Every time I download a video

**Expected behavior**
I should be able to download and keep scrolling with no issues

**Video**
https://youtube.com/shorts/HwoLE0iqWVI?feature=share

**Comment by adushaj** [comment]
Yep, same issue on iPhone 12 Pro. Not only does the app crash after downloading a video, it will mark any posts you read as unread.
